src/sentry/backup/imports.py

```diff
@@ -232,20 +232,34 @@ def do_writes(pk_map: PrimaryKeyMap) -> None:
 
             do_write(pk_map, model_name, json_data)
 
+    # Resolves slugs for all imported organization models via the PrimaryKeyMap and reconciles
+    # their slug globally via control silo by issuing a slug update.
+    def resolve_org_slugs_from_pk_map(pk_map: PrimaryKeyMap):
+        from sentry.services.organization import organization_provisioning_service
+
+        org_model_name = get_model_name(Organization)
+        org_pk_mapping = pk_map.mapping[str(org_model_name)]
```

Comment:
```suggestion
        org_pk_mapping = pk_map.partition(org_model_name))
```

Maybe. Not 100% sure if that makes it clearer or not.

Reply:
ahh yep! I'll use that, thanks!